contributors/devel/pull-requests.md

```diff
@@ -90,17 +90,10 @@ To see how to format your release notes, view the [PR template](https://github.c
 
 Release notes apply to PRs on the master branch. For cherry-pick PRs, see the [cherry-pick instructions](cherry-picks.md). The only exception to these rules is when a PR is not a cherry-pick and is targeted directly to the non-master branch.  In this case, a `release-note-*` label is required for that non-master PR.
 
-**Labels**
```

Comment:
I figured we probably don't want to be pushing people to edit labels manually anymore.